scripts/us_census/acs5yr/subject_tables/s2201/README.md

```diff
@@ -0,0 +1,28 @@
+# Importing Census ACS5Year Table S2201
+
+## Input Data
+
+The data is downloaded from
+`https://www.census.gov/acs/www/data/data-tables-and-tools/subject-tables/` and
```

Comment:
A TODO to have a script to download a given subject table.  Or you can point it out to Ajai and BLR team on how you did, so they can put together a general script.

Reply:
updated the script to download directly from the website -- however this will require finding the download_id (which will be an input param to the script) I will update the readme with this info